scalyr_agent/builtin_monitors/shell_monitor.py

```diff
@@ -139,6 +141,8 @@ def gather_sample(self):
             output += "\n"
         output += stdout_text
 
+        output = six.ensure_text(output)
```

Comment:
I believe it makes more sense for us to convert output from bytes to unicode instead of doing the opposite.

All the config options which are specified by the end user (think extract pattern, etc.) will always be string / unicode and not bytes type so it makes sense we work with that type in those places.

Reply:
Agreed.